tests/unit/test_protocol_errors.py

```diff
@@ -0,0 +1,105 @@
+"""
```

Comment:
Ah, the error codes are not going to be part of the ga4gh protocol, I guess?  That gives us a lot more flexibility then... although from a specification standpoint I'm not sure if that's the best idea...